web/src/routes/SurveyPages/SurveyPage.tsx

```diff
@@ -22,31 +23,34 @@ const Container = styled.div`
 export default function SurveyPage() {
   const { id } = useParams();
   const [theme, toggleTheme] = useTheme();
-  const [surveyData, setSurveyData] = useState<SurveyResponse>();
 
-  const fetchSurveyData = async () => {
-    try {
-      const request: AxiosResponse<SurveyResponse> = await axios.get<SurveyResponse>(requests.getSurvey + id);
-
-      setSurveyData(request.data);
-    } catch (error) {
-      const { name } = error as unknown as AxiosError;
-      // TODO: handle error while fetching data
-    }
-  };
-
-  useEffect(() => {
-    fetchSurveyData();
-  }, []);
+  // FIXME: cacheTime and staleTime is appropriate?
+  const { data, isLoading, isError, error } = useQuery<SurveyResponse>(['survey', id], fetchSurveyData, {
+    cacheTime: 3000,
+    staleTime: 60000,
```

Comment:
Same reason as above. But please modify this to

```typescript
cacheTime: 15 * 60 * 1000, // 15 minutes
staleTime: 10 * 60 * 1000, // 10 minutes
```

Would each survey take more than 10 min? I wish this service to provide some simple, easy-to-answer questions(max 10 questions I hope)